Upstream-Status: Inappropriate [oe-core specific]

TESTS_64 is empty, install will fail due to missing file operand

Signed-off-by: Ting Liu <b28495@freescale.com>

--- a/tests/Makefileold	2013-10-11 22:54:12.073313123 -0500
+++ b/tests/Makefile	2013-10-11 22:55:36.070305004 -0500
@@ -293,6 +293,7 @@
 	$(INSTALL) -m 755 $(HELPERS:%=obj64/%) $(DESTDIR)$(INST_TESTSDIR64)/obj64
 	$(INSTALL) -m 755 $(HELPER_LIBS:%=obj64/%) $(DESTDIR)$(INST_TESTSDIR64)/obj64
 #	$(INSTALL) -m 755 $(TESTS_64:%=obj64/%) $(DESTDIR)$(INST_TESTSDIR64)/obj64
+	$(INSTALL) -m 755 $(TESTS_64_STATIC:%=obj64/%_static) $(DESTDIR)$(INST_TESTSDIR64)/obj64
 	$(INSTALL) -m 755 run_tests.py $(DESTDIR)$(INST_TESTSDIR64)
 
 install: $(OBJDIRS:%=%/install)
